tests/test_Config.py

```diff
@@ -250,12 +250,12 @@ def testSave(self):
 
         del roundTrip
         # test saving to an open file
-        outfile = open("roundtrip.test", "w")
-        self.comp.saveToStream(outfile)
-        outfile.close()
+        with open("roundtrip.test", "w") as outfile:
+            self.comp.saveToStream(outfile)
```

Comment:
Add tests for `saveToString` too?

Reply:
There isn't actually a `saveToString`, just a `loadFromString`, though I suppose symmetry suggests I should add it; done, along with tests.